ref/glossary.md

```diff
@@ -248,6 +269,11 @@ One of the main objects of Anbox Cloud. Every time you launch an application or
 
 See https://discourse.ubuntu.com/t/about-containers/17763.
 
+<a name="control-node"></a>
+### Control node
```

Comment:
Is this `Control node` the management node that we described below? If so, let's be consistent in the way we use terms on this page.